src/test/java/com/volasoftware/tinder/controllers/RegistrationControllerIntegrationTests.java

```diff
@@ -0,0 +1,80 @@
+package com.volasoftware.tinder.controllers;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.volasoftware.tinder.constants.Gender;
+import com.volasoftware.tinder.dtos.AccountDTO;
+import com.volasoftware.tinder.dtos.RegisterDTO;
+import com.volasoftware.tinder.services.contracts.AccountService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WebMvcTest(RegisterController.class)
+public class RegistrationControllerIntegrationTests {
```

Comment:
Keep same name as tested class appending {optional: Integration}Test as suffix. For me, a name like RegisterControllerTest is enough. 
The decision for class, method naming  should be already made and documented somewhere and applied for consistency.

Reply:
class was renamed